libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -141,6 +142,19 @@ userdata Vector2f operator +
 userdata Vector2f operator -
 userdata Vector2f method copy Vector2f
 
+userdata Quaternion field q1 float'skip_check read write
```

Comment:
needs docs, generate with `./waf copter --scripting-docs` Then copy out the new ones and put them into the old docs. Bonus points for also adding useful param names and descriptions.

Reply:
Will do, cheers.

Reply:
This seems interesting. What does it do exactly?